pkgs/applications/science/medicine/aliza/default.nix

```diff
@@ -3,24 +3,25 @@
 with lib;
 stdenv.mkDerivation {
   pname = "aliza";
-  version = "1.98.43";
+  version = "1.98.57";
   src = fetchurl {
     # See https://www.aliza-dicom-viewer.com/download
-    url = "https://drive.google.com/uc?export=download&id=1HiDYUVN30oSWZWt3HBp7gNRBCLLtJM1I";
-    sha256 = "0d70q67j2q0wdn4m2fxljqb97jwmscqgg3rm1rkb77fi2ik206ra";
+    url = "https://drive.google.com/uc?export=download&id=1-AXa3tjy_onecW2k7ftjAQl0KGTb0B1Y";
```

Comment:
can we upload the version to web.archive.org?

Reply:
Added the web.archive.org upload as an alternative download.